### PR TITLE
changes in package json and updated index html

### PR DESCRIPTION
**Pull request by ibrahim777764**
### Milestone: Hello Microverse

In this milestone we,

- Created  branches with different base branches.
- Understood the concept of version control.
- Followed Gitflow.
- Opened pull requests in GitHub.
- Wrote descriptive, easy to understand commit messages.
- Wrote short, detailed, easy to understand descriptions of pull requests.
- Wrote a descriptive and easy to understand README in business English for every project submitted.
- Were demonstrate an ability to submit a project for code review.

### CODE REVIEW

**Comment by ibrahim777764** [comment]
please review my code and let me know if there are any changes to do,THANKYOU

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <b>Hello Microverse</b>
```
Comment (micheaol) [inline index.html]: Please ensure you use `h1` tag for the `Hello Microverse` according to the requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ibrahim777764 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
- _Check the comments under the review._
- Please ensure you create test.md file and this file should be ignored by git.
- Please ensure you always add a short summary of changes introduced in the pull request. Please check this [LINK](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for help.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ibrahim777764** [comment]
i done every thing as you suggested ,please have a look is every thing is fine now,thankyou

File: index.html (hunk 1 above)
Comment (micheaol) [reply index.html]: @ibrahim777764  Kindly re-submit for re-review once you are done with all the required changes.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ibrahim777764 , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

 ## Required Changes ♻️
- Please ensure you add a short summary of changes introduced in the pull request as earlier requested. Please check this [LINK](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for help.
## Example: 

pre

## Yours:
you
tions
## Optional sugges


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ibrahim777764** [comment]
hope so   every thing is done as you suggested

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @ibrahim777764 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**